userguide/system.rst

```diff
@@ -790,7 +790,7 @@ command line.
 By default, SEDs are not locked until the administrator takes ownership
 of them. Ownership is taken by explicitly configuring a global or
 per-device password in the %brand% |web-ui| and adding the password to
-the SEDs.
+the SEDs. This also allows %brand% to create :ref:`pools` with SEDs.
```

Comment:
Not clear what _This_ refers to here.  The added sentence is kind of specific, implying that other things might be possible without it.  I think that what this is really trying to say is that when ownership/passwords for SEDs are entered into %brand%, it can use them without manual unlocking.